index.html

```diff
@@ -5,9 +5,62 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="styles.css" />
-    <title>Hello Microverse</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Libre+Barcode+39+Text&family=Poppins:wght@200;400;600&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;1,300;1,400;1,500&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Libre+Barcode+39+Text&family=Playfair+Display&family=Poppins:wght@200;400;600&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;1,300;1,400;1,500&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="https://www.pngplay.com/wp-content/uploads/12/Astro-Boy-Transparent-Images.png"
+    />
+    <title>AstroboyReloaded | Portfolio</title>
   </head>
   <body>
-    <h1 class="title">Hello Microverse!</h1>
+    <header>
+      <nav>
+        <a
+          href="#"
+          class="logo"
+          role="img"
+          aria-label="Logo: Right Triangle and rhombus"
+        ></a>
```

Comment:
- [OPTIONAL] You did well implementing the required design :+1: but as a  project requirement you should add these images only using `css`. Also, this is not the logo so you shouldn't use `<a>` tag because it is not going to link to a specific part of your file.